src/transport_none.h

```diff
@@ -87,6 +89,14 @@ shmem_transport_probe(void)
     return;
 }
 
+static inline
+shmem_transport_addr_t shmem_transport_get_local_addr(void)
+{
+    shmem_transport_addr_t addr = {0};
```

Comment:
This will cause all PEs to appear to be on the same node, which might be wrong.  A better solution might be to define the addr type as:
```
typedef int shmem_transport_addr_t;
```
So, here you can answer this query as:
```
return shmem_internal_my_pe;
```
This says all PEs are one different nodes.  Although, you could imagine a user wanting only shared memory and not being interested in the network, in which case, you could return the Unix hostname here.  That might be better, if it's not too hard to implement.